spec/unit/parser_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "dry/web/parser"
+
+RSpec.describe Dry::Web::Parser do
+  let(:klass) { Dry::Web::Parser }
+  let(:parser) { klass.new(file) }
+  let(:file) { SPEC_ROOT.join("fixtures/test/.env.test") }
+
+  describe "#parse" do
+    subject { parser.parse }
```

Comment:
I don't mind using `subject(:settings)` to indicate that this is the object we're evaluating our tests against (n.b. it should still be referred to via `settings` rather than as an anonymous subject)